homeassistant/components/squeezebox/media_player.py

```diff
@@ -221,16 +221,17 @@ class SqueezeBoxEntity(MediaPlayerEntity):
     _last_update: datetime | None = None
     _attr_available = True
 
-    def __init__(self, player: Player) -> None:
+    def __init__(self, player: Player, server: Server) -> None:
         """Initialize the SqueezeBox device."""
-        self._player = player
+        self._player: Player = player
```

Comment:
```suggestion
        self._player = player
```
I think this is inferred now